dvc/command/version.py

```diff
@@ -28,12 +28,11 @@ def run(self):
         from dvc.repo import Repo
 
         info = [
-            f"DVC version: {__version__}",
-            f"Python version: {platform.python_version()}",
-            f"Platform: {platform.platform()}",
-            f"Binary: {is_binary()}",
-            f"Package: {PKG}",
```

Comment:
@efiop, is `Binary` necessary? I assume we can deduct that through `PKG`, right?

Reply:
@skshetry Yeah, could totally live without `Binary`. PKG is enough.